tests/test_redshift.py

```diff
@@ -353,17 +353,18 @@ def test_redshift_snapshot_mark(self):
                     {
                         "type": "value",
                         "key": "SnapshotIdentifier",
-                        "value": "c7n-snapshot",
+                        "value": "c7n-test-snapshot",
                     }
                 ],
                 "actions": [{"type": "mark-for-op", "days": 30, "op": "delete"}],
             },
-            session_factory=factory,
+            session_factory=factory, config={'account_id': '644160558196'}
         )
 
         resources = p.run()
         self.assertEqual(len(resources), 1)
-        arn = p.resource_manager.generate_arn(resources[0]["SnapshotIdentifier"])
+        arn = p.resource_manager.generate_arn(
```

Comment:
same as below re get_arns instead of manual id gen

Reply:
Addressed